src/main/java/AlexExceptions.java

```diff
@@ -0,0 +1,8 @@
+public class AlexExceptions extends Exception {
```

Comment:
The class AlexExceptions extends Exception, but the name AlexExceptions is too generic and does not clearly describe what kind of exceptions it handles. 😠